### DISCUSSION TITLE
About output a text with timestamp.

### DISCUSSION THREAD

**Discussion by Huowuge** [SWivid/F5-TTS#437] [Q&A]
I didn’t find how to set up TTS to output the original text with a timestamp, which would facilitate real-time subtitle creation and speech alignment. Could you please tell me where this function is set?

**Comment by SWivid** [comment]
F5 has no explicit text-speech force-alignment, so it might not be realized with itself, though could use some ASR tools like whisper to get the timestamp.

**Reply by Huowuge** [reply]
ok, thanks
